keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/TestController.kt

```diff
@@ -0,0 +1,21 @@
+package com.netflix.spinnaker.keel.rest
+
+import com.netflix.spinnaker.keel.yaml.APPLICATION_YAML_VALUE
+import com.netflix.spinnaker.kork.exceptions.SystemException
+import org.springframework.http.MediaType.APPLICATION_JSON_VALUE
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping(path = ["/test"])
+class TestController {
```

Comment:
I'm not quite sure if we need it in main, but if you can add some comments here about the purpose of this controller, it will be great!

Reply:
The purpose of the controller is to make it easier to test out that alerts are configured correctly. For example, if we have an alert on exceptions, or 500 errors, we can test it out by inducing the error and then checking if the alert fired.

The reason I'd like to have it in main is that I want to be able to test that the alert is actually working correctly in the main environment. I worry about making a mistake configuring an alert for the main environment which would result in it never firing.

Reply:
Oh I see, make sense to me! If you can just add this to the actual code (for future reference, and for other developers to know), it will be great.

Reply:
Consider adding a @ConditionalOnProperty("tests.error-simulation:false") that you can override in config to enable the endpoint in specific environments. I'm a little uncomfortable just leaving this always on everywhere.

Reply:
@gal-yardeni @luispollo Those are good ideas. I'll add comments in the code and protect it with a config value.

Reply:
Addressed in 595a47f.